packages/contracts/test/framework/plugin/plugin-repo.ts

```diff
@@ -3,25 +3,28 @@
 
 import {expect} from 'chai';
 import {ethers} from 'hardhat';
+import {ContractFactory} from 'ethers';
 import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
 import {keccak256, solidityPack} from 'ethers/lib/utils';
 
 import {
   PluginRepo,
+  PluginRepo__factory,
   PluginUUPSUpgradeableSetupV1Mock,
   PlaceholderSetup__factory,
   TestPlugin__factory,
   IERC165__factory,
   IPluginRepo__factory,
   IProtocolVersion__factory,
-  UUPSUpgradeable__factory,
 } from '../../../typechain';
+import {PluginRepo__factory as PluginRepo_V1_0_0__factory} from '../../../typechain/@aragon/osx-v1.0.1/framework/plugin/repo/PluginRepo.sol';
+
+import {ozUpgradeCheckManagingContract} from '../../test-utils/uups-upgradeable';
```

Comment:
Unrelated, there is a function at the end of the page: tagHash()
can we move it to a better place?